--- node_modules/openid-client/lib/passport_strategy.js	2023-06-23 13:47:39.749175747 -0500
+++ node_modules/openid-client/lib/passport_strategy_log_error.js	2023-06-26 13:51:44.419894855 -0500
@@ -189,6 +189,7 @@
     this._verify(...args);
     /* end authentication response */
   })().catch((error) => {
+    console.log(error);
     if (
       (error instanceof OPError &&
         error.error !== 'server_error' &&
